tests/integration/test_dependencies_compatibility_protobuf.py

```diff
@@ -183,7 +183,7 @@ async def test_protobuf_schema_compatibility_dependencies(registry_async_client:
         json={"schemaType": "PROTOBUF", "schema": evolved_schema, "references": evolved_references},
     )
     assert res.status_code == 200
-    assert res.json() == {"is_compatible": False}
+    assert not res.json().get("is_compatible")
```

Comment:
issue: not its true also for `None` that its the default value of the `.get` when its empty

Reply:
Good catch, fixed